R/qenv-concat.R

```diff
@@ -0,0 +1,50 @@
+#' Concatenate two `qenv` objects
+#'
+#' Combine two `qenv` objects by simple concatenate their environments and the code.
+#' We recommend to use the `join` method for most cases.
```

Comment:
```suggestion
#' We recommend to use the `join` method to have a stricter control 
#' in case `x` and `y` contain duplicated bindings and code.
```